test/dotnet/utils/TestUtils.cs

```diff
@@ -39,10 +39,50 @@ public bool insertBulkCopy(DbConnection bblCnn, DbCommand bblCmd, String sourceT
 			DbDataReader reader = null;
 			try
 			{
+				/* To Enforce Reset Connection. */
 				reader = bblCmd.ExecuteReader();
-				SqlBulkCopy bulkCopy = new SqlBulkCopy(ConfigSetup.BblConnectionString);
+				using (SqlConnection destinationConnection =
```

Comment:
how does this reset connection?

Reply:
pooling=false is removed from connection string and .net driver does not reset connection if we only provide this connectionString to the SqlBulkCopy API, but if we manually create and open this connection object then it will reset connection. Check PR description for validation through logging